packages/storybook/src/spright/chat/chat-matrix.stories.ts

```diff
@@ -0,0 +1,138 @@
+import type { StoryFn, Meta } from '@storybook/html';
+import { html, ViewTemplate } from '@microsoft/fast-element';
+import { chatMessageTag } from '../../../../spright-components/src/chat/message';
+import { ChatMessageType } from '../../../../spright-components/src/chat/types';
+import {
+    createMatrix,
+    sharedMatrixParameters,
+    createMatrixThemeStory
+} from '../../utilities/matrix';
+import { createStory } from '../../utilities/storybook';
+import { hiddenWrapper } from '../../utilities/hidden';
+import { loremIpsum } from '../../utilities/lorem-ipsum';
+import { richTextViewerTag } from '../../../../nimble-components/src/rich-text/viewer';
+import { chatConversationTag } from '../../../../spright-components/src/chat/conversation';
+
+const chatMessageTypes = [
+    ['outbound', ChatMessageType.outbound],
+    ['inbound', ChatMessageType.inbound],
+    ['system', ChatMessageType.system]
+] as const;
+type ChatMessageTypeState = (typeof chatMessageTypes)[number];
+
+const metadata: Meta = {
+    title: 'Tests Spright/Chat Message',
```

Comment:
We can align the tests name with the docs name

```suggestion
    title: 'Tests Spright/Chat',
```

Reply:
Made the change as suggested.